### PR TITLE
Add Codecov for code coverage to Travis

### PR DESCRIPTION
**Pull request by ibrahimsharaf**
Using Code Coverage to determine how much code is being tested.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@ scipy
 phe
 line_profiler
 clint
+coverage
```
Comment (baldassarreFe) [inline requirements.txt]: Hey, I think `line_profiler`, `clint`, `coverage`, `pytest` and `pytest-flake8` should be moved to `test-requirements`. While `sklearn` should go all the way to `dev-requirements`

Comment (ibrahimsharaf) [reply requirements.txt]: Where are the dev-requirements?

Comment (baldassarreFe) [reply requirements.txt]: In `dev-requirements.txt`, but maybe you have to pull from `master` because it's been added quite recently by #270

Comment (ibrahimsharaf) [reply requirements.txt]: @baldassarreFe done.

**Pull request merged**